src/UI/Component/Glyph/Factory.php

```diff
@@ -675,5 +675,28 @@ public function sad($action = null);
 	 */
 	public function angry($action = null);
 
-
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The Attachment glyph indicates that a file is attached or can be attached to an object or entity.
+	 *   composition: >
+	 *      The Attachment Glyph uses the glyphicon-paperclip.
+	 *   effect: >
+	 *       Clicking executes an action which delivers these attachments to the actor or initiates a process to add new attachments.
+	 *
+	 * context: Impose the information whether or not files have been attached to emails in the folder view.
+	 *
+	 * rules:
+	 *   composition:
+	 *       1: >
+	 *          A Status Counter MAY indicate the overall amount of attachments.
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be 'Attachment'.
```

Comment:
Is this implemented? If not, it seems also no being implemented for the other glyphs, right? Currently I fail to see where in the template the label is rendered.

Reply:
@klees : label is passed through constructor in implementation and rendered in line 37 in renderer:

`		$tpl->setVariable("LABEL", $this->txt($component->getAriaLabel()));
`

Note that this is done by using the glyph.context and glyph.standard.html